src/components/DefaultEpic.tsx

```diff
@@ -56,51 +61,70 @@ const imageStyles = css`
-                heard, explore where others turn away, and rigorously challenge those in power.
-            </p>
-            <p className={bodyStyles}>
-                We hope you will consider supporting us today. We need your support to keep
-                delivering quality journalism that’s open and independent. Every reader
-                contribution, however big or small, is so valuable.{' '}
-                <strong className={highlightWrapperStyles}>
-                    <span className={highlightStyles}>
-                        Support The Guardian from as little as £1 - and it only takes a minute.
-                        Thank you.
-                    </span>
-                </strong>
-            </p>
-            <div className={buttonWrapperStyles}>
-                <CallToAction
-                    url="https://support.theguardian.com/uk/contribute"
-                    linkText="Support The Guardian"
-                />
-                <img
-                    src="https://assets.guim.co.uk/images/acquisitions/2db3a266287f452355b68d4240df8087/payment-methods.png"
-                    alt="Accepted payment methods: Visa, Mastercard, American Express and PayPal"
-                    className={imageStyles}
-                />
-            </div>
-        </section>
-    );
+type Props = {
+    heading: string | undefined;
+    paragraphs: string[];
+    highlighted: string[];
+};
+
+type HighlightedProps = {
+    highlighted: string[];
+};
+
+type BodyProps = {
+    paragraphs: string[];
+    highlighted: string[];
 };
+
+const Highlighted: React.FC<HighlightedProps> = ({ highlighted }: HighlightedProps) => (
+    <strong className={highlightWrapperStyles}>
+        {' '}
+        {highlighted.map((highlightText, idx) => (
+            <span
+                key={idx}
+                className={highlightStyles}
+                dangerouslySetInnerHTML={{ __html: replacePlaceholders(highlightText) }}
+            />
+        ))}
+    </strong>
+);
+
+const Body: React.FC<BodyProps> = ({ highlighted, paragraphs }: BodyProps) => (
+    <>
+        {paragraphs.length > 0 &&
+            paragraphs.map((paragraph, idx) => (
+                <p key={idx} className={bodyStyles}>
+                    <span dangerouslySetInnerHTML={{ __html: replacePlaceholders(paragraph) }} />
+
+                    {highlighted.length > 0 && idx === paragraphs.length - 1 && (
+                        <Highlighted highlighted={highlighted} />
+                    )}
+                </p>
+            ))}
+    </>
+);
+
+export const DefaultEpic: React.FC<Props> = ({ heading, paragraphs, highlighted }: Props) => (
+    <section className={wrapperStyles}>
+        {heading && (
+            <h2
+                className={headingStyles}
+                dangerouslySetInnerHTML={{ __html: replacePlaceholders(heading) }}
+            />
+        )}
+
+        <Body paragraphs={paragraphs} highlighted={highlighted} />
```

Comment:
Is it worth using a different name here because of confusion with the `body` HTML element?